spec/breadcrumbs/breadcrumb_spec.rb

```diff
@@ -0,0 +1,47 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+require 'bugsnag/breadcrumbs/breadcrumb'
+
+describe Bugsnag::Breadcrumbs::Breadcrumb do
+  describe "initialize" do
+    it "should assign arguments correctly" do
+      breadcrumb = Bugsnag::Breadcrumbs::Breadcrumb.new("my message", "test type", {:a => 1, :b => 2}, :manual)
```

Comment:
is auto not a boolean?

Reply:
Yes, it will be a boolean inside the class, but not the leave_breadcrumbs method. I'll update that.